sfaira/versions/metadata/base.py

```diff
@@ -347,17 +347,24 @@ def get_descendants(self, node: str) -> List[str]:
         node = self.convert_to_id(node)
         return list(networkx.descendants(self.graph, node))
 
-    def is_a(self, query: str, reference: str) -> bool:
+    def is_a(self, query: str, reference: str, convert_to_id=True) -> bool:
         """
         Checks if query node is reference node or an ancestor thereof.
 
         :param query: Query node name. Node ID or name.
         :param reference: Reference node name. Node ID or name.
+        :param convert_to_id: If True -> apply self.convert_to_id to query and reference input
         :return: If query node is reference node or an ancestor thereof.
         """
-        query = self.convert_to_id(query)
-        reference = self.convert_to_id(reference)
-        return query in self.get_ancestors(node=reference) or query == reference
+        if convert_to_id:
+            query = self.convert_to_id(query)
+            reference = self.convert_to_id(reference)
+        else:
+            if isinstance(query, str):
+                query = [query]
+
+        ancestors = self.get_ancestors(node=reference)
+        return np.any(query in ancestors) or (reference in query)
```

Comment:
This needs to be `return np.any([x in ancestors for x in query]) or (reference in query)`

Reply:
I removed the vectorization from the is_a function again. So this is fine now